fastai/optimizer.py

```diff
@@ -340,14 +340,22 @@ def set_item_pg(pg, k, v):
 # Cell
 pytorch_hp_map = {'momentum': 'mom', 'weight_decay': 'wd', 'alpha': 'sqr_mom', 'betas__0': 'mom', 'betas__1': 'sqr_mom'}
 
+# Cell
+def convert_params(o:list) -> list:
+    splitter = []
+    for group in o:
+        if isinstance(group, dict): return o
```

Comment:
Why is this returned rather than added to `splitter`? Can you add a comment explaining what's happening?